### ISSUE TITLE
0005562:
Add custom fields to all apps

### ISSUE THREAD

**Issue by Gloirin** [tine20/tine20#2836]
**Reported by OCMoe on 7 Feb 2012 19:00**

**Version:** Milan (2012-03) Beta 2

Given that adding custom field support to any given model is dead easy (I&#039;m doing it in my local copy for the Crm and Sales apps), I&#039;d love to see custom field support in every app. I can add it by myself, no problem there, but it would be nice to have that officially.

**Comment by Gloirin** [comment]
Related to #3093

**Comment by Gloirin** [comment]
Depends on #3967

**Comment by Gloirin** [comment]
Depends on #4039

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 8 Feb 2012 14:13**

if you like, you can become a tine 2.0 developer (see http://www.tine20.org/wiki/index.php/Become_a_developer_/_CLA) and commit this to the project :)

**Comment by pschuele** [comment]
already implemented, i think.
